src/patternfly/components/InputGroup/examples/InputGroup.md

```diff
@@ -40,14 +40,14 @@ cssPrefix: pf-c-input-group
 {{/input-group}}
 <br>
 {{#> input-group}}
-  {{#> dropdown id="dropdown-example-collapsed1" dropdown--HasToggleIcon="true"}}
-    {{#> dropdown-toggle-text}}
-      Dropdown
-    {{/dropdown-toggle-text}}
-  {{/dropdown}}
-  {{#> form-control controlType="input" input="true" form-control--attribute='type="text" id="textInput3" name="textInput3" aria-label="Input with dropdown and button" aria-describedby="inputDropdownButton1"'}}
+  {{#> select id="select-example-collapsed1" select--HasToggleIcon="true" select--attribute='style="width: 100px;"'}}
```

Comment:
`dropdown--HasToggleIcon` is unique to the dropdown component. If you look at the [select component examples](https://github.com/patternfly/patternfly/blob/v4/src/patternfly/components/Select/examples/Select.md), you'll see none of them use `select--HasToggleIcon`, so you don't need that here.

```suggestion
  {{#> select id="select-example-collapsed1" select--attribute='style="width: 100px;"'}}
```

Reply:
OK I just hit the "Commit suggestion" button